components/crud-web-apps/jupyter/frontend/i18n/fr/messages.fr.xlf

```diff
@@ -407,6 +473,7 @@
       </trans-unit>
       <trans-unit id="7a084ea40b1b4313d812cfc6c7f41333f5021c45" datatype="html">
         <source> IfNotPresent </source>
+        <target>Si elle n'est pas présente</target>
```

Comment:
Is this used as a value? In which case it probably needs to stay IfNotPresent. Otherwise the translation is valid.